src/AudioDevice.cpp

```diff
@@ -22,6 +23,11 @@ indie::AudioDevice::~AudioDevice()
     CloseAudioDevice();
 }
 
+void indie::AudioDevice::initAudio()
+{
+    InitAudioDevice();
```

Comment:
It would be better if there was a verification to check if audiodevice wasn't already initialized.